core/vtk/ttkSignedDistanceField/ttkSignedDistanceField.cpp

```diff
@@ -0,0 +1,262 @@
+  const auto domain = vtkDataSet::GetData(inputVector[0]);
+  double bounds[6];
+  domain->GetBounds(bounds);
+  DataExtent[1] = xResolution_;
+  DataExtent[3] = yResolution_;
+  DataExtent[5] = zResolution_;
+  int shift = (ExpandBox ? 2 : 0);
+  spacing_[0] = (bounds[1] - bounds[0]) / (DataExtent[1] - 1 - shift);
+  spacing_[1] = (bounds[3] - bounds[2]) / (DataExtent[3] - 1 - shift);
+  spacing_[2] = (bounds[5] - bounds[4]) / (DataExtent[5] - 1 - shift);
+  for(unsigned int i = 0; i < spacing_.size(); ++i)
+    invSpacingSquared_[i] = 1.0 / spacing_[i] / spacing_[i];
+  std::stringstream ss;
+  ss << "Resolution : " << DataExtent[1] << ", " << DataExtent[3] << ", "
+     << DataExtent[5];
+  printMsg(ss.str());
+  ss.str("");
+  ss << "Spacing : " << spacing_[0] << ", " << spacing_[1] << ", "
+     << spacing_[2];
+  printMsg(ss.str());
+  ss.str("");
+  ss << "Extent : " << spacing_[0] * xResolution_ << ", "
+     << spacing_[1] * yResolution_ << ", " << spacing_[2] * zResolution_;
+  printMsg(ss.str());
+  Origin[0] = bounds[0];
+  Origin[1] = bounds[2];
+  Origin[2] = bounds[4];
+  if(ExpandBox) {
+    Origin[0] -= spacing_[0];
+    Origin[1] -= spacing_[1];
+    Origin[2] -= spacing_[2];
+  }
+}
+
+int ttkSignedDistanceField::RequestData(vtkInformation *ttkNotUsed(request),
+                                        vtkInformationVector **inputVector,
+                                        vtkInformationVector *outputVector) {
+  using ttk::SimplexId;
+
+  const auto domain = vtkDataSet::GetData(inputVector[0]);
+
+  if(!domain)
+    return !this->printErr("Unable to retrieve required input data object.");
+
+  // triangulation Domain
+  auto triangulation = ttkAlgorithm::GetTriangulation(domain);
+  if(!triangulation) {
+    this->printErr("Input triangulation pointer is NULL.");
+    return -1;
+  }
+
+  this->preconditionTriangulation(triangulation);
+
+  if(triangulation->isEmpty()) {
+    this->printErr("Triangulation allocation problem.");
+    return -2;
+  }
+
+  if(!domain) {
+    this->printErr("Input pointer is NULL.");
+    return -3;
+  }
+
+  const auto numberOfVertices = domain->GetNumberOfPoints();
+  if(numberOfVertices <= 0) {
+    this->printErr("Domain has no points.");
+    return -4;
+  }
+
+  std::stringstream ss;
+  ss << "Surface number of vertices : " << triangulation->getNumberOfVertices();
+  printMsg(ss.str());
+
+  ss.str("");
+  ss << "Surface number of triangles : "
+     << triangulation->getNumberOfTriangles();
+  printMsg(ss.str());
+
+  // Create encompassing triangulation
+  computeOutputInformation(inputVector);
+  vtkSmartPointer<vtkImageData> imageData
+    = vtkSmartPointer<vtkImageData>::New();
+  imageData->SetOrigin(Origin[0], Origin[1], Origin[2]);
+  imageData->SetSpacing(spacing_[0], spacing_[1], spacing_[2]);
+  imageData->SetDimensions(DataExtent[1], DataExtent[3], DataExtent[5]);
+
+  auto encompassingTriangulation = ttkAlgorithm::GetTriangulation(imageData);
```

Comment:
`encompassingTriangulation` -> `boundingTriangulation` (?)
(thanks for adjusting the other occurrences)

Reply:
Done in the next commit!